### PR TITLE
Added Muslim Aid bake sale fundraising event

### PR DESCRIPTION
**Pull request by RalphLynch**
## Describe your changes
I added the recent bake sale for Muslim Aid to the fundraising section, and updated the fundraising total accordingly.
## Checklist before requesting a review
- [x] I have performed a self-review of my changes
- [x] I have made corresponding changes to the documentation and comments where neccessary
- [x] If the changes affect the website, I have tested the branch using `bundle exec jekyll serve` on my local machine

### CODE REVIEW

**Comment by svejnohatomas** [comment]
@RalphLynch Can you please assign yourself as an assignee and Nathan or me to the PRs as reviewers? We won't see it otherwise. Thanks

File: _data/sitetext.yml [hunk 1]
```diff
@@ -105,6 +105,10 @@ en: &DEFAULT_EN
         desc: "We held a charity pub quiz alongside some other societies and raised **£1140** for Islamic Relief UK, a Muslim charity providing humanitarian aid for the devastating earthquake that affected Turkey and Syria in January 2023."
         image: assets/img/timeline/islamicrelief.png
         alt: "Islamic Relief"
+      - title: "Muslim Aid"
+        desc: "We held a bakesale on campus with the Islamic and Pakistani societies and raised **£731.25** for Muslim Aid and their Palestine appeal, to help with the humanitarian crisis in Gaza."
+        image: assets/img/timeline/muslimaid.png
+        alt: "Muslim Aid"
     end: "And many <br> more!"
     total_funds_raised: 6415.00 # amount of funds raised in GBP
```
Comment (svejnohatomas) [inline _data/sitetext.yml]: @RalphLynch Can you please also update the total? Thanks

**Review by svejnohatomas — CHANGES_REQUESTED** [review]

**Comment by RalphLynch** [comment]
Hi, sorry I took a while to get back to you, I've just been a bit busy with coursework. Also, apologies for not assigning you to the review earlier, it must have slipped my mind while making the PR.

**Review by svejnohatomas — APPROVED** [review]

**Comment by svejnohatomas** [comment]
@RalphLynch Thank you. Approved.

**Pull request merged**